src/core/a-scene.js

```diff
@@ -93,24 +89,12 @@ var AScene = module.exports = registerElement('a-scene', {
         var resizeCanvas = this.resizeCanvas.bind(this);
         this.setupKeyboardShortcuts();
         this.attachFullscreenListeners();
```

Comment:
I'm a bit confused. Are you suggesting that *if vr-mode is forced to VR*, then ignore fullscreen events? And how should we allow vr-mode being forced to VR?